HomeWork10/HomeWork10/Services/RoomService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using HomeWork10.Models;
+using HomeWork10.Services.Abstractions;
+using System.Runtime.CompilerServices;
+
+namespace HomeWork10.Services
+{
+    internal class RoomService : IRoomService
+    {
+        private readonly IRosetteService _serviceRosette;
+        private readonly Room _room;
+
+        public RoomService(IRosetteService rosetteService)
+        { 
+            _serviceRosette = rosetteService;
+            _room = new Room(); 
+        }
+       
+        public void TurnOnOffApliance(ElectricalAppliances electrical)                              
+        {
+            Console.WriteLine($"Connect to rosette {electrical.Id} : {electrical.GetType().Name} y/n?");
+           
+            _room.Appliances.Add(_serviceRosette.СonnecDisconectDevice(electrical, (Console.ReadKey().Key == ConsoleKey.Y)));
+        }
+
+        private double GetTotalElectricityCost()
+        {
+            return _room.Appliances.Where(p=>p.TornOn == true)
```

Comment:
_room.Appliances.Where(p=>p.TornOn) you dont need "== true" here

Reply:
i add to apliance to rosette and turn on but i have some to turn off)) thats whay  i check for on